doc/Manuals/LoadBalancing/ig-xlb_x-road_external_load_balancer_installation_guide.md

```diff
@@ -0,0 +1,785 @@
+Set owner of the `recovery.conf` to `postgres:postgres`, mode `0600`
+
+Modify `postgresql.conf`:
+
+```
+ssl = on
+ssl_ca_file   = '/etc/xroad/postgresql/ca.crt'
+ssl_cert_file = '/etc/xroad/postgresql/server.crt'
+ssl_key_file  = '/etc/xroad/postgresql/server.key'
+
+listen_addresses = localhost
+
+# no need to send WAL logs
+# wal_level = minimal
+# max_wal_senders = 0
+# wal_keep_segments = 0
+
+hot_standby = on
+hot_standby_feedback = on
+```
+Notice that on RHEL, during `pg_basebackup` the `postgresql.conf` was copied from the master node so the WAL sender
+parameters should be disabled. Also check that `listen_addresses` is localhost-only.
+
+Start the database instance
+
+**RHEL:**
+```bash
+systemctl start postgresql-serverconf
+```
+**Ubuntu:**
+```bash
+/etc/init.d/postgresql start
+```
+Note that on Ubuntu, the command starts all configured database instances.
+
+## 5. Configuring data replication with rsync over SSH
+
+### 5.1 Set up SSH between slaves and the master
+On the master, set up a system user that can read `/etc/xroad`. A system user has their password disabled and can not log
+in normally.
+
+**Ubuntu:**
+
+```bash
+adduser --system --ingroup xroad xroad-slave
+```
+**RHEL:**
+
+```bash
+useradd -r -m -g xroad xroad-slave
+```
+
+
+Create an `.ssh` folder and the authorized keys file:
+```bash
+sudo mkdir -m 755 -p /home/xroad-slave/.ssh && sudo touch /home/xroad-slave/authorized_keys
+ ```
+**Warning:**  The owner of the file should be `root` and `xroad-slave` should not have write permission to the file.
+
+On the slave nodes, create an ssh key (`ssh-keygen`) without a passphrase for the `xroad` user and add the public keys in
+the `/home/xroad-slave/.ssh/authorized_keys` of the master node. To finish, from slave nodes, connect to the master host
+using `ssh` and accept the host key.
+
+### 5.2 Set up periodic configuration synchronization on the slave nodes
+
+The following configuration, which will be set up on the slave nodes will synchronize the configuration in `/etc/xroad`
+periodically (once per minute) and before the services are started. That means that during boot, if the master server is
+available, the configuration will be synchronized before the `xroad-proxy` service is started. If the master node is down,
+there will be a small delay before the services are started.
+
+> Note that only modifications to the signer keyconf will be applied when the system is running. Other configuration changes
+> require restarting the services, which is not automatic.
+
+#### 5.2.1 RHEL: Use `systemd` for configuration synchronization
+
+Add `xroad-sync` as a `systemd` service.
+
+Create a new file `/etc/systemd/system/xroad-sync.service`:
+```
+[Unit]
+Description=X-Road Sync Task
+After=network.target
+Before=xroad-proxy.service
+Before=xroad-signer.service
+Before=xroad-confclient.service
+Before=xroad-jetty.service
+[Service]
+User=xroad
+Group=xroad
+Type=oneshot
+Environment=XROAD_USER={{ xroad_slave_ssh_user }}
```

Comment:
should explain which parts to fill in (remove curly brackets also) since reader might not be familiar with ansible template syntax